### PR TITLE
P1/feat/88 handle mobile dropdown

### PR DESCRIPTION
**Pull request by netlight-johannadagfalk**
PR for both story 93- remove scroll in background when mobile kanban is open and story 88- toggle between globe and hamburger menu.

### CODE REVIEW

File: app/assets/main/react_components/dashboard/climate_actions/manage_actions/dropdown-kanban/MobileKanbanContainer.jsx [hunk 1]
```diff
@@ -11,10 +13,25 @@ const MobileKanbanContainer = ({ children, userActions }) => {
...
+  const toggleScroll = (bool) => {
+    const scrollable = document.getElementById("scrollable");
+    bool
+      ? (scrollable.className += " overflow-hidden")
+      : (scrollable.className -= " overflow-hidden");
+  };
+
+  useEffect(() => {
+    const toggleMobileKanban = () => setShowMobileKanban(false);
+    const hamburgerMenu = document.getElementById("nav-toggler");
```
Comment (netlight-saraandersson) [inline app/assets/main/react_components/dashboard/climate_actions/manage_actions/dropdown-kanban/MobileKanbanContainer.jsx:25]: Maybe use a different name than hamurgerMenu, maybe "mobileDropDownMenu?"

**Review by netlight-saraandersson — APPROVED** [review]
Looks good! Left a comment regarding naming :D

File: app/assets/main/react_components/dashboard/climate_actions/manage_actions/dropdown-kanban/MobileKanbanContainer.jsx (hunk 1 above)
Comment (netlightebba) [reply app/assets/main/react_components/dashboard/climate_actions/manage_actions/dropdown-kanban/MobileKanbanContainer.jsx:25]: I think we would avoid using "mobile" in component names and in variables (except the ones that check if mobile). The components and variables should be independent and when they are rendered should be controlled by the media check. As Simon said once, the component only happens to be used when mobile, it should not depend on it. :)

Comment (netlightebba) [reply app/assets/main/react_components/dashboard/climate_actions/manage_actions/dropdown-kanban/MobileKanbanContainer.jsx:25]: By this I also imply that we should change name on the component itself 🙈

**Review by netlightebba — APPROVED** [review]

**Pull request merged**